jsonnetfile.lock.json

```diff
@@ -1,125 +1,114 @@
 {
-  "dependencies": [
-    {
-      "name": "grafana-builder",
-      "source": {
-        "git": {
-          "remote": "https://github.com/grafana/jsonnet-libs",
-          "subdir": "grafana-builder"
+    "dependencies": [
+        {
+            "name": "grafana-builder",
+            "source": {
+                "git": {
+                    "remote": "https://github.com/grafana/jsonnet-libs",
+                    "subdir": "grafana-builder"
+                }
+            },
+            "version": "8f9d72b2e35b5f3cc1b7c2a8af9bbae7658804e2"
+        },
+        {
+            "name": "grafonnet",
+            "source": {
+                "git": {
+                    "remote": "https://github.com/grafana/grafonnet-lib",
+                    "subdir": "grafonnet"
+                }
+            },
+            "version": "cb9e43f59558ff6338a76ae1806a0fb4b70b1b16"
+        },
+        {
+            "name": "jaeger-mixin",
+            "source": {
+                "git": {
+                    "remote": "https://github.com/jaegertracing/jaeger/",
+                    "subdir": "monitoring/jaeger-mixin"
+                }
+            },
+            "version": "85b2d2a625a5e73eacc5cb81054cb357cbcc4877"
+        },
+        {
+            "name": "kube-thanos",
+            "source": {
+                "git": {
+                    "remote": "https://github.com/thanos-io/kube-thanos",
+                    "subdir": "jsonnet/kube-thanos"
+                }
+            },
+            "version": "4c3ab85bc3cebe5c227e6037948f9ffa4c50715d"
+        },
+        {
+            "name": "ksonnet",
+            "source": {
+                "git": {
+                    "remote": "https://github.com/ksonnet/ksonnet-lib",
+                    "subdir": ""
+                }
+            },
+            "version": "0d2f82676817bbf9e4acf6495b2090205f323b9f"
+        },
+        {
+            "name": "slo-libsonnet",
+            "source": {
+                "git": {
+                    "remote": "https://github.com/metalmatze/slo-libsonnet",
+                    "subdir": "slo-libsonnet"
+                }
+            },
+            "version": "101618191a5dae835f7c8a90e6f0df40072e2cb3"
+        },
+        {
+            "name": "telemeter",
+            "source": {
+                "git": {
+                    "remote": "https://github.com/openshift/telemeter",
+                    "subdir": "jsonnet/telemeter"
+                }
+            },
+            "version": "8dae8893776e237688d7ff39821265a4f467f213"
+        },
```

Comment:
You just need to commit this change only, and then run the `./build.sh`.

Reply:
@LiliC I updated the `version`, but there was no value to replace the existing `sum` with.

Reply:
After attempting to merge the change I got this error:
> jb: error: failed to install packages: checksum mismatch for telemeter. Expected ks15FK8j+RCu6QpmgI6Ofiy/RZ8eLQVss4VNo/IWQts= but got tLjALazakKQ6Z0VtG74rX/TLJo4Jh7lvN6fkXHWJxNc=

I'll update the sum to match it.

Reply:
What do you mean merge the change?

Reply:
Sorry - I updated the PR to include `tLjALazakKQ6Z0VtG74rX/TLJo4Jh7lvN6fkXHWJxNc=` as the sum.